packages/db-ra-data-rest/src/index.js

```diff
@@ -0,0 +1,153 @@
+        );
+      }
+      return {
+        data: json,
+        total: parseInt(headers.get("x-total-count").split("/").pop(), 10),
+      };
+    });
+  },
+
+  getOne: (resource, params) =>
+    httpClient(`${apiUrl}/${resource}/${params.id}`).then(({ json }) => ({
+      data: json,
+    })),
+
+  getMany: (resource, params) => {
+    const query = {
+      "where.id.in": params.ids.join(","),
+    };
+    const url = `${apiUrl}/${resource}?${stringify(query)}`;
+    return httpClient(url).then(({ json }) => ({ data: json }));
+  },
+
+  getManyReference: (resource, params) => {
+    const { page, perPage } = params.pagination;
+    const { field, order } = params.sort;
+
+    const query = {
+      ...formatFilters(params.filter),
+      [`where.${params.target}.eq`]: params.id,
+      [`orderby.${field}`]: order.toLowerCase(),
+      limit: perPage,
+      offset: (page - 1) * perPage,
+      totalCount: true,
+    };
+    const url = `${apiUrl}/${resource}?${stringify(query)}`;
+
+    return httpClient(url).then(({ headers, json }) => {
+      if (!headers.has("x-total-count")) {
+        throw new Error(
+          "The X-Total-Count header is missing in the HTTP Response. The jsonServer Data Provider expects responses for lists of resources to contain this header with the total number of results to build the pagination. If you are using CORS, did you declare X-Total-Count in the Access-Control-Expose-Headers header?"
+        );
+      }
+      return {
+        data: json,
+        total: parseInt(headers.get("x-total-count").split("/").pop(), 10),
+      };
+    });
+  },
+
+  update: (resource, params) =>
+    httpClient(`${apiUrl}/${resource}/${params.id}`, {
+      method: "PUT",
+      body: JSON.stringify(params.data),
+    }).then(({ json }) => ({ data: json })),
+
+  // platformatic doesn't handle filters on UPDATE route, so we fallback to calling UPDATE n times instead
+  updateMany: (resource, params) =>
+    Promise.all(
+      params.ids.map((id) =>
+        httpClient(`${apiUrl}/${resource}/${id}`, {
+          method: "PUT",
+          body: JSON.stringify(params.data),
+        })
+      )
+    ).then((responses) => ({ data: responses.map(({ json }) => json.id) })),
+
+  create: (resource, params) =>
+    httpClient(`${apiUrl}/${resource}`, {
+      method: "POST",
+      body: JSON.stringify(params.data),
+    }).then(({ json }) => ({
+      data: { ...params.data, id: json.id },
+    })),
+
+  delete: (resource, params) =>
+    httpClient(`${apiUrl}/${resource}/${params.id}`, {
+      method: "DELETE",
+    }).then(({ json }) => ({ data: json })),
+
+  // platformatic doesn't handle filters on DELETE route, so we fallback to calling DELETE n times instead
```

Comment:
We actually support it but only in GraphQL. Would you mind opening an issue to expose this to openapi too?